Rakefile

```diff
@@ -3,6 +3,13 @@ require 'suse/connect'
 require 'rspec/core/rake_task'
 require 'date'
 
+module MonkeypatchRakeLastComment
```

Comment:
This is the error in Prophet: http://river.suse.de/job/SCC-Connect/144/console (`method missing` at the very bottom).

The `last_comment` method has been silently removed from Rake 11.0.1, then restored with a deprecation warning: https://github.com/ruby/rake/blob/v11.1.2/lib/rake/task_manager.rb#L9-L12

We currently happen to use the affected 11.0.x versions in our build pipeline, so this is really needed.
I'll add that as a code comment.